2018-11-14-Continuous-Integration.md

```diff
@@ -1,264 +1,15 @@
 ---
-layout: blog
+layout: redirect
+redirect: https://unicorn-utterances.com/posts/travis-ci-for-android/
 title: "Continuous Integration with Travis CI for Android"
 description: "An in-depth tutorial explaining how to set up Travis CI to deploy signed builds to Google Play. Among other things."
 tags:
   - android
   - instructional
   - ci
-author: "James Fenn"
+author: "James Fenn / Unicorn Utterances"
```

Comment:
This change is sort of unnecessary - the current [post layout](https://github.com/doubledotlabs/DoubleDotLabs.github.io/blob/master/_includes/item-post.html#L4) appends the domain name to the title if a post is redirected.

Reply:
Gotcha. I was trying to follow the README 😅